### DISCUSSION TITLE
Add output to rubocop telling the user how many files have violations in them

### DISCUSSION THREAD

**Discussion by mjhymowitz** [rubocop/rubocop#11912] [Ideas]
My idea is to add a command that returns a little more statistical information for users to understand whats going on in their code or add another line of output when a user runs 

`$ rubocop` or `$ rubocop -review`
output> "Out of #{Total_files} files #{Files_In_Violation} files have violations".

**Comment by noraj** [comment]
Also maybe how many files are violating a cop and how many violations there are per cop.

Update: I just saw https://github.com/rubocop/rubocop/discussions/11911
